src/lib/stores/wallet.ts

```diff
@@ -4,8 +4,17 @@ import { getFromLocalStorage, removeFromLocalStorage, saveToLocalStorage } from
 import { writable, get } from 'svelte/store'
 import type { Writable } from 'svelte/store'
 
+export interface Token {
+	name: string
+	symbol: string
+	amount: bigint
+	decimals: number
+	image: string
+}
+
 export interface WalletState {
 	wallet?: HDNodeWallet
+	balances?: Token[]
```

Comment:
Maybe it's better not mixing optionals with arrays, because the empty array can also express optionality, but the two cases has to be handled differently in the code.